mappings/net/minecraft/advancement/Advancement.mapping

```diff
@@ -16,6 +25,7 @@ CLASS net/minecraft/unmapped/C_kdwyuhdb net/minecraft/advancement/Advancement
 		ARG 1 style
 	METHOD m_egdqigyx createTask ()Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
 	METHOD m_ehjzgofc getId ()Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD m_ggypmlkv sendsTelemetryEvent ()Z
```

Comment:
```suggestion
	METHOD m_ggypmlkv doesSendTelemetryEvent ()Z
```
to follow the [boolean getter convension](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#getters-setters-withers-and-creators)